wiki/Options/en.md

```diff
@@ -344,84 +341,58 @@ This section is about chat, spectators, multi, and osu!direct.
-| `Prefer no-video downloads`                | (For [osu!supporters](/wiki/osu!supporter/) only) osu!direct downloads will be confined to no-video versions of beatmaps.        | Checkbox | `Disabled` |
+| Name | Effect | Type | Default |
+| --- | --- | --- | --- |
+| `Integrate with Yahoo! status display` | Your Yahoo! messenger will show the song you are currently playing or listening to. This needs to be set from the website. | Checkbox | `Disabled` |
+| `Integrate with MSN Live status display` | Your Windows Live Messenger will show the song you are currently playing or listening to. This needs to be set from the osu!website. | Checkbox | `Disabled` |
+| `Automatically start osu!direct downloads` | For [osu!supporters](/wiki/osu!supporter/) only. When spectating or multiplaying, the beatmap will be downloaded automatically. | Checkbox | `Enabled` |
+| `Prefer no-video downloads` | For [osu!supporters](/wiki/osu!supporter/) only. osu!direct downloads will be confined to no-video versions of beatmaps. | Checkbox | `Disabled` |
 
 ### In-Game Chat
 
-| Name                                          | Effect                                                | Type   | Default    |
-|-----------------------------------------------|-------------------------------------------------------|--------|------------|
-| `Filter offensive words`                      | Replace offensive words with `*beep*`.                | Button | `Disabled` |
-| `Filter foreign characters`                   | Removes any non-standard ASCII characters.            | Button | `Disabled` |
-| `Log private messages`                        | Private messages will be saved to the `osu!/` folder. | Button | `Disabled` |
-| `Block private messages from non-friends`     | Enabling this will confine PMs to friends only.       | Button | `Disabled` |
-| `Chat ignore list (space-separated list)`     | Words you put here will be ignored.                   | Text   | _(Empty)_  |
-| `Chat highlight words (space-separated list)` | Words you put here will be highlighted in chat.       | Text   | _(Empty)_  |
+| Name | Effect | Type | Default |
+| --- | --- | --- | --- |
+| `Filter offensive words` | Replace offensive words with `*beep*`. | Button | `Disabled` |
+| `Filter foreign characters` | Removes any non-standard ASCII characters. | Button | `Disabled` |
+| `Log private messages` | Private messages will be saved to the `osu!/Logs` folder. | Button | `Disabled` |
+| `Block private messages from non-friends` | Enabling this will confine private messages to friends only. | Button | `Disabled` |
+| `Chat ignore list (space-separated list)` | Words you put here will be ignored. | Text | *(empty)* |
+| `Chat highlight words (space-separated list)` | Words you put here will be highlighted in chat. | Text | *(empty)* |
 
-## Maintenance
+## Maintance
```

Comment:
The old spelling is the correct spelling. The image somewhat has the wrong reference and wrong file name at the same time.